test/functional/readconcern_tests.js

```diff
@@ -121,12 +186,13 @@ describe('ReadConcern', function() {
           client =
             test.urlOptions != null
               ? configuration.newClient(url, options)
-              : configuration.newClient({ w: 1 }, { poolSize: 1, readConcern: test.readConcern });
+              : configuration.newClient(url);
```

Comment:
Now that this is simplified, can we simplify the check for `test.urlOptions` above? It seems like we could make an `options` object that includes any options we need to pass in to `newClient` and conditionally add `readConcern: { level: test.level } ` to them if `test.urlOptions` is present. Then we could change this into `client = configuration.newClient(url, options)`.